src/backend/XmlHttpRequest.ts

```diff
@@ -31,6 +33,10 @@ export interface XmlHttpRequestOptions {
   // Used as the URL prefix for fetched files.
   // Default: Fetch files relative to `url`.
   baseUrl?: string;
+  // Whether to use Fetch or XmlHttpRequest.
+  // Can be 'xhr', 'fetch', or 'auto'.
+  // Default: 'auto' which uses fetch if its available, and XHR as a fallback
```

Comment:
You can stick with the boolean for now. I can do the fancy class maneuver if it makes sense post-merge.